core/src/main/scala/org/apache/spark/rpc/RpcAddress.scala

```diff
@@ -23,30 +23,37 @@ import org.apache.spark.util.Utils
 /**
  * Address for an RPC environment, with hostname and port.
  */
-private[spark] case class RpcAddress(_host: String, port: Int) {
-
-  lazy val host: String = Utils.addBracketsIfNeeded(_host)
+private[spark] case class RpcAddress(host: String, port: Int) {
 
   def hostPort: String = host + ":" + port
 
   /** Returns a string in the form of "spark://host:port". */
   def toSparkURL: String = "spark://" + hostPort
 
   override def toString: String = hostPort
+
+
```

Comment:
Could you revert this empty line addition?

Reply:
Sure :)